styles.css

```diff
@@ -0,0 +1,418 @@
+  color: white;
+  font-weight: 700;
+  margin-top: 3%;
+}
+
+.short-orange-hr {
+  width: 5%;
+  height: 3px;
+  opacity: 1;
+  background-color: #ec5242;
+}
+
+#main-program ul {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+}
+
+#mobile-open-nav li {
+  padding: 1% 0;
+}
+
+#main-program ul > li {
+  background-color: rgba(255, 255, 255, 0.1);
+  align-items: center;
+  justify-content: space-between;
+  margin: 10px 3px;
+  height: 200px;
+}
+
+#main-program ul > li > * {
+  padding: 10px;
+}
+
+#main-program a {
+  display: none;
+}
+
+.course-item {
+  align-items: center;
+}
+
+.course-item h3 {
+  padding-left: 2px;
+}
+
+#main-program ul > li > p {
+  color: white;
+}
+
+.course-item-description {
+  width: 50%;
+}
+
+#hr-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+#hr-wrapper hr {
+  width: 5%;
+  height: 2px;
+  opacity: 1;
+  background-color: #ec5242;
+}
+
+#main-program img {
+  width: 70px;
+  height: 70px;
+}
+
+#main-program button {
+  width: 60%;
+  height: 90px;
+  background-color: #ec5242;
+  color: white;
+  border: 0;
+  margin: 5%;
+}
+
+#featured-speakers {
+  padding-top: 10%;
+}
+
+#featured-speakers ul > li {
+  display: flex;
+}
+
+#featured-speakers ul > li > img {
+  height: 150px;
+  width: 150px;
+}
+
+#featured-speakers ul > li > * {
+  padding: 2%;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+.speaker-hr {
+  width: 5%;
+  opacity: 1;
+  height: 2px;
+}
+
+#more-button-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+#featured-speakers button {
+  width: 70%;
+  height: 60px;
+  border: 1px solid grey;
+  font-weight: bold;
+  margin-bottom: 40px;
+  background-color: white;
+}
+
+.nav {
+  display: none;
+}
+
+#partners {
+  display: none;
+}
+
+footer {
+  display: none;
+}
+
+.display-none {
+  display: none;
+}
+
+#mobile-open-nav ul {
+  padding: 1% 7%;
+  background-color: #272a31;
+}
+
+#mobile-open-nav a {
+  text-decoration: none;
+  color: #ec5242;
+}
+
+.hide-on-mobile {
+  display: none !important;
+}
+
+#close-button {
+  margin-left: 100%;
+}
+
+.large-font {
+  font-size: 35px;
+}
+
+.webinar-date {
+  font-weight: bolder;
+}
+
+#webinar-date-subtitle {
+  color: #7e7e7e;
+  font-size: 25px;
+}
+
+@media screen and (min-width: 768px) {
+  #hamburger {
+    display: none;
+  }
+
+  .nav {
+    display: block;
+  }
+
+  nav ul {
+    display: flex;
+  }
+
+  #upper-nav {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    background-color: #272a31;
+  }
+
+  #upper-nav li {
+    padding: 5px;
+    font-size: 15px;
+  }
+
+  #upper-nav i,
+  #upper-nav span {
+    color: white;
+  }
+
+  #lower-nav li {
+    padding: 20px 7%;
+    font-size: 15px;
+  }
+
+  #main-program ul {
+    padding: 0 10%;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  #main-program ul > li {
+    flex-direction: column;
+    justify-content: flex-start;
+    height: 230px;
+    font-size: 10px;
+  }
```

Comment:
- [ ] Please fix the view in this scale see the screenshot 👇

![hhh](https://user-images.githubusercontent.com/61562626/132569159-9e1c0471-ad39-4f4f-b8a8-2bb81b693ec3.jpeg)